docs/source/getting_started/installation.md

```diff
@@ -87,3 +102,33 @@ Known labextensions:
         @jupyterlab/toc v4.0.0  enabled  OK
         nbdime-jupyterlab v2.0.0  enabled  OK
 ```
+
+### Docker 
+
+If you have Docker installed, you can install and use JupyterLab and Elyra by selecting one of the [ready-to-run Docker images](https://hub.docker.com/r/elyra/elyra/tags) maintained by the Elyra Team:
+
+ - `elyra/elyra:latest` has the latest released version installed.
+ - `elyra/elyra:x.y.z` has version `x.y.z` installed.
+ - `elyra/elyra:dev` is automatically re-built each time a change is committed to the master branch.
+
+The command below starts the most recent development build in a clean environment:
+
+```
+docker run -it -p 8888:8888 elyra/elyra:dev jupyter lab --debug
+```
+
+To make a local directory containing your Notebooks (e.g. `${HOME}/opensource/jupyter-notebooks/`) available in your Docker container, you can use a mount command similar to the following:
+
+```
+docker run -it -p 8888:8888 -v ${HOME}/opensource/jupyter-notebooks/:/home/jovyan/work -w /home/jovyan/work elyra/elyra:dev jupyter lab --debug
+```
+
```

Comment:
How about adding a new section to mount the available metadata from local system to docker? Just follow the same pattern above and point to `jupyter --data-dir/metadata`

Reply:
As suggested this doesn't work unless I am missing something:

```
docker run -it -p 8888:8888 -v $HOME/jupyter-notebooks/:/home/jovyan/work -w /home/jovyan/work -v $HOME/jupyter-metadata:/home/jovyan/.local/share/jupyter/metadata elyra/elyra:dev jupyter lab --debug
```

yields

```
[D 16:23:07.010 LabApp] Searching ['/home/jovyan/work', '/home/jovyan/.jupyter', '/opt/conda/etc/jupyter', '/usr/local/etc/jupyter', '/etc/jupyter'] for config files
[D 16:23:07.011 LabApp] Looking for jupyter_config in /etc/jupyter
[D 16:23:07.011 LabApp] Looking for jupyter_config in /usr/local/etc/jupyter
[D 16:23:07.011 LabApp] Looking for jupyter_config in /opt/conda/etc/jupyter
[D 16:23:07.011 LabApp] Looking for jupyter_config in /home/jovyan/.jupyter
[D 16:23:07.011 LabApp] Looking for jupyter_config in /home/jovyan/work
[D 16:23:07.013 LabApp] Looking for jupyter_notebook_config in /etc/jupyter
[D 16:23:07.014 LabApp] Loaded config file: /etc/jupyter/jupyter_notebook_config.py
[D 16:23:07.014 LabApp] Looking for jupyter_notebook_config in /usr/local/etc/jupyter
[D 16:23:07.014 LabApp] Looking for jupyter_notebook_config in /opt/conda/etc/jupyter
[D 16:23:07.015 LabApp] Loaded config file: /opt/conda/etc/jupyter/jupyter_notebook_config.json
[D 16:23:07.015 LabApp] Looking for jupyter_notebook_config in /home/jovyan/.jupyter
[D 16:23:07.015 LabApp] Loaded config file: /home/jovyan/.jupyter/jupyter_notebook_config.py
[D 16:23:07.015 LabApp] Looking for jupyter_notebook_config in /home/jovyan/work
Traceback (most recent call last):
  File "/opt/conda/lib/python3.7/site-packages/traitlets/traitlets.py", line 528, in get
    value = obj._trait_values[self.name]
KeyError: 'runtime_dir'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/conda/bin/jupyter-lab", line 8, in <module>
    sys.exit(main())
  File "/opt/conda/lib/python3.7/site-packages/jupyter_core/application.py", line 268, in launch_instance
    return super(JupyterApp, cls).launch_instance(argv=argv, **kwargs)
  File "/opt/conda/lib/python3.7/site-packages/traitlets/config/application.py", line 663, in launch_instance
    app.initialize(argv)
  File "</opt/conda/lib/python3.7/site-packages/decorator.py:decorator-gen-7>", line 2, in initialize
  File "/opt/conda/lib/python3.7/site-packages/traitlets/config/application.py", line 87, in catch_config_error
    return method(app, *args, **kwargs)
  File "/opt/conda/lib/python3.7/site-packages/notebook/notebookapp.py", line 1766, in initialize
    self.init_configurables()
  File "/opt/conda/lib/python3.7/site-packages/notebook/notebookapp.py", line 1380, in init_configurables
    connection_dir=self.runtime_dir,
  File "/opt/conda/lib/python3.7/site-packages/traitlets/traitlets.py", line 556, in __get__
    return self.get(obj, cls)
  File "/opt/conda/lib/python3.7/site-packages/traitlets/traitlets.py", line 535, in get
    value = self._validate(obj, dynamic_default())
  File "/opt/conda/lib/python3.7/site-packages/jupyter_core/application.py", line 99, in _runtime_dir_default
    ensure_dir_exists(rd, mode=0o700)
  File "/opt/conda/lib/python3.7/site-packages/jupyter_core/utils/__init__.py", line 13, in ensure_dir_exists
    os.makedirs(path, mode=mode)
  File "/opt/conda/lib/python3.7/os.py", line 221, in makedirs
    mkdir(name, mode)
PermissionError: [Errno 13] Permission denied: '/home/jovyan/.local/share/jupyter/runtime'
```

I can only get it to work if the mount point is `jupyterlab --data-dir` (`/home/jovyan/.local/share/jupyter` in the image):

```
docker run -it -p 8888:8888 -v $HOME/jupyter-notebooks/:/home/jovyan/work -w /home/jovyan/work -v $HOME/jupyter-metadata:/home/jovyan/.local/share/jupyter elyra/elyra:dev jupyter lab --debug
```

Reply:
I updated the documentation to mount the JupyterLab data directory.

Reply:
you are actually right, as the current image does not have the `jupyter data-dir` created... as you mentioned I could make the following to work:
```
docker run -it -p 8888:8888 -v $HOME/jupyter-notebooks/:/home/jovyan/work -w /home/jovyan/work -v $HOME/Library/Jupyter:/home/jovyan/.local/share/jupyter elyra/elyra:1.1.0 jupyter lab --debug

```